lib/Core/Executor.cpp

```diff
@@ -1322,10 +1322,14 @@ Executor::toConstant(ExecutionState &state,
     return CE;
 
   ref<ConstantExpr> value;
-  bool success =
-      solver->getValue(state.constraints, e, value, state.queryMetaData);
-  assert(success && "FIXME: Unhandled solver failure");
-  (void) success;
+  if (seedMap.count(&state))
+    value = getValueFromSeeds(seedMap[&state], e);
+  else {
```

Comment:
I would assume that `getValueFromSeeds` might not return a constant.
Do a check for `ConstantExpr` and return early, otherwise we would end up with lots of "silently concretizing" messages. which is misleading and annoying.

Reply:
If not returned, invoke the solver.

Reply:
Done